src/main/java/mate/academy/spring/controller/AuthenticationController.java

```diff
@@ -0,0 +1,22 @@
+package mate.academy.spring.controller;
+
+import mate.academy.spring.model.dto.request.UserRequestDto;
+import mate.academy.spring.security.AuthenticationService;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class AuthenticationController {
+    private final AuthenticationService authenticationService;
+
+    public AuthenticationController(AuthenticationService authenticationService) {
```

Comment:
`Optional` if you decided to use Lombok, you could remove constructors using a specific annotation